src/k8s/console.ts

```diff
@@ -27,8 +27,8 @@ export class Console extends OpenShiftItem {
             const consoleUrl = `${spec.port.targetPort}://${spec.host}`;
             return consoleUrl;
         } else {
-            const consoleUrl = await Console.odo.execute(Command.showServerUrl());
-            return consoleUrl.stdout;
+            const getUrlObj = await Console.odo.execute(Command.showConsoleUrl());
+            return JSON.parse(getUrlObj.stdout).data.consoleURL;
```

Comment:
This change is specific to 3.x cluster and not 4.x cluster. In the issue you have mentioned that the 4.x cluster does not work properly.

Reply:
okay @mohitsuman I will check.

Reply:
old code works only when we login with kubeadmin but it doesn't work when we login with `oc login -u developer -p developer https://api.crc.testing:6443` for 4.x cluster.

now it works for both kubeadmin and for developer.